snippets/oadp-mtc-operator.adoc

```diff
@@ -0,0 +1,13 @@
+//This snippet appears in the following assemblies:
```

Comment:
This. needs :_content-type: SNIPPET.
[https://docs.google.com/document/d/1o4rrsgowUzGiVSH84Wg-Tt4DxmDXE_hg2H946NnXj6s/edit](https://docs.google.com/document/d/1o4rrsgowUzGiVSH84Wg-Tt4DxmDXE_hg2H946NnXj6s/edit)